app/layout/project/sidebar/project/inventory-panel/features/list/index.tsx

```diff
@@ -88,6 +87,16 @@ export const ProjectFeatureList = (): JSX.Element => {
     [dispatch, visibleFeatures]
   );
 
+  const isShown = useCallback(
```

Comment:
Do not use `useCallback` to compute values, you have `useMemo` for that. Although, I don't think it's needed in this case.